.github/workflows/pr-highlight-changes.yml

```diff
@@ -11,14 +11,14 @@ jobs:
               uses: actions/checkout@v3
             - name: Install prerequisites
               run: |
-                  npm install -g pnpm@^6.24.2
+                  npm install -g pnpm
```

Comment:
This is the only place left where we install pnpm separately. I thought about removing it but there are so many changes already that I'll do this in a follow up later.